MBBSEmu.Tests/ExportedModules/Majorbbs/strtol_Tests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace MBBSEmu.Tests.ExportedModules.Majorbbs
+{
+    public class strtol_Tests : ExportedModuleTestBase
+    {
+        private const int STRTOL_ORDINAL = 586;
+
+        [Theory]
+        [InlineData("123456 Hello some text", 7, 10, 123456)]
+        [InlineData("2345678 YESSS", 7, 10, 2345678)]
+        [InlineData("78 A", 7, 10, 78)]
+        [InlineData("0x23CACE A very big number", 1, 16, 2345678)]
```

Comment:
let's do a test with an empty string as well as a string that isn't actually a number, i.e. all text/non-digit

Reply:
And a negative number